### PR TITLE
AB Email Notifications - Time fix for dropdowns

### PR DESCRIPTION
**Pull request by HasnatL-Adapptlimited**
# Links 
[Base PR](https://github.com/ReliefApplications/ems-frontend/pull/2460)

Backend PR Related:
[Backend PR](https://github.com/ReliefApplications/ems-backend/pull/1034)

# Tickets

- [ ] Dropdown and Radio buttons showing date

### CODE REVIEW

**Comment by AntoineRelief** [comment]
@HasnatL-Adapptlimited 
for this PR and future PRs, please add more description to bug fix or link to existing ticket, or screenshots
the current description doesn't help to understand what is exactly the issue the PR is fixing

**Comment by AntoineRelief** [comment]
@HasnatL-Adapptlimited 

Can you help me understand this PR?

With the current description I don't understand what it is supposed to solve

**Comment by HasnatL-Adapptlimited** [comment]
Hey @AntoineRelief 

We were noticing that for some radio, dropdown fields it was converting them to date even though they were text like "Item 2"

image

How its actually supposed to look:

image

File: libs/shared/src/lib/components/email/email.service.ts [hunk 1]
```diff
@@ -208,6 +194,32 @@ export class EmailService {
     });
   }
 
+  /**
+   * Constructs the EmailService instance.
```
Comment (AntoineRelief) [inline libs/shared/src/lib/components/email/email.service.ts:198]: @HasnatL-Adapptlimited 
FYI for future commits
Avoid comments of constructor to describe what the constructor does ( avoid to use "constructs" )
Prefer to use the same jsdoc comment than the one you put on top of the service or component

And for params, avoid to start using "the" as a prefix
Also, no need to have long texts like that for service injections
for example:
apollo -> Apollo service
http -> Http client
restService -> Shared rest service

**Comment by AntoineRelief** [comment]
@HasnatL-Adapptlimited 

merged 👍 

Will now proceed with a deployment on UAT

**Pull request merged**